Makefile

```diff
@@ -1,17 +1,17 @@
 ############################# Main targets #############################
-# Install all tools and builds binaries.
-install: update-tools proto bins
-
-# Run all possible checks and tests.
-all: proto check bins test
-
 # Rebuild binaries.
-bins: clean-bins proto temporal-server tctl temporal-cassandra-tool temporal-sql-tool
+bins: clean-bins temporal-server tctl temporal-cassandra-tool temporal-sql-tool
+
+# Install all tools, recompile proto files, run all possible checks and tests (long but comprehensive).
+all: update-tools clean proto bins check test
 
 # Delete all build artefacts.
-clean: clean-bins clean-proto clean-test-results
+clean: clean-bins clean-test-results
+
+# Recompile proto files.
+proto: clean-proto install-proto-submodule protoc fix-proto-path proto-mock
 
-# Update proto submodule from remote and rebuild proto files.
+# Update proto submodule from remote and recompile proto files.
```

Comment:
What is the command to regenerate the proto files?

Reply:
`make proto`. I moved it to this "header" section.